docs/clients/scheduler.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: client
+category: clients
+name: Scheduler
+package: async-aws/scheduler
+---
+
+## Usage
+
+### Create a schedule group
+
+```php
+use AsyncAws\Scheduler\Input\CreateScheduleGroupInput;
+use AsyncAws\Scheduler\SchedulerClient;
+
+$scheduler = new SchedulerClient();
+
+$result = $scheduler->createScheduleGroup(new CreateScheduleGroupInput([
+    'ClientToken' => 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+    'Name' => 'foo',
```

Comment:
Could you please use "realistic" values (liske a real usecase of  the service) in the documentation? It helps to understand how it works